### PR TITLE
update LABELS.md

### PR DESCRIPTION
**Pull request by tejaswari7**
fix [Update Labels for Issues and PRs] #10649

### CODE REVIEW

File: doc/LABELS.md [hunk 1]
```diff
@@ -1,38 +1,39 @@
-# Labels used at Public Lab
-At Public Lab, we use many different labels and tags to keep up with issues and pull requests.
++# Labels used at Public Lab :label:
++At Public Lab, we use labels to categorize and keep up with issues and pull requests.
```
Comment (TildaDares) [inline doc/LABELS.md:2]: @tejaswari7 Can you remove the `+` signs in front of the new lines you added? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Review by TildaDares — APPROVED** [review]
Thanks @tejaswari7! Great job

**Comment by tejaswari7** [comment]
thank you!! @TildaDares for guiding me

**Review by govindgoel — APPROVED** [review]
LGTM!

**Comment by TildaDares** [comment]
Thanks everyone!

**Pull request merged**